perun-core/src/main/resources/schema.sql

```diff
@@ -857,7 +857,7 @@ create table tasks (
 	status varchar(16) not null,
 	start_time timestamp,
 	end_time timestamp,
-	engine_id integer not null,
+	engine_id integer,
```

Comment:
engine_id in tasks_results can not be null...there must be an engine that processed given task to obtain task result.